server/common/database/migrations/20191113102651-alter-user-update-role-values.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const { role: NEW_ROLES } = require('../../../../common/config');
```

Comment:
You should specify `NEW_ROLES` explicitly since this file might change over time which would make this migration invalid.
As a rule of thumb migrations shouldn't rely on some specific version of codebase since it makes it mutable.